cfme/tests/infrastructure/test_host.py

```diff
@@ -248,3 +248,68 @@ def test_250_vmware_hosts_loading(appliance, create_250_hosts, view_type):
     view.entities.paginator.set_items_per_page(1000)
     view.toolbar.view_selector.select(view_type)
     wait_for(view.entities.get_first_entity, timeout=60, message='Wait for the view')
+
+
+@test_requirements.general_ui
+@pytest.mark.provider([RHEVMProvider], selector=ONE, scope="module", override=True)
```

Comment:
Is Vmware different? Curious why just RHEV here.

Reply:
No particular reason. But it was explicitly mentioned in the test to use RHEV. But I guess there's no harm in adding other infra providers.

Reply:
Hey @jawatts, I've modified the test to use both infra providers.

Reply:
@valaparthvi Thanks!